### ISSUE TITLE
sanity-check error when Makefile not world-readable

### ISSUE THREAD

**Issue by yasuhirokimura** [freebsd/poudriere#584]
Host OS: FreeBSD 11.1-RELEASE amd64
Poudriere version: 3.2.4
Revision of ports tree: 459482

Update of `mail/postfix-policyd-spf-python` was submitted to FreeBSD Bugzilla with following URL:

https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=220706

Proposed patch was updated several times and latest one is compatible with flavors support of ports tree. It can be built and installed successfully on real FreeBSD 11.1-RELEASE amd64 environment. But on poudriere with jail of same OS version it fail to be built with check-sanity phase error. 

Attached file below is build log of poudriere with 11.1-RELEASE amd64 jail.

[py36-postfix-policyd-spf-python-2.0.2.log](https://github.com/freebsd/poudriere/files/1648678/py36-postfix-policyd-spf-python-2.0.2.log)

**Comment by bdrewery** [comment]
I committed your PR earlier as I could not reproduce any problem.
I didn't see your check-sanity error in any of the bugzilla bugs but I could have missed it. The problem is that your Makefile has bad permissions.  Poudriere in 3.2 now builds several phases for the port as *not root*. This particular error can happen if the Makefile is not readable by world.

I will rework this bug to making Poudriere be more clear about this particular issue. It's become a common confusion.

**Comment by yasuhirokimura** [comment]
> The problem is that your Makefile has bad permissions. 

You are right! Permissions of mail/postfix-policyd-spf-python/Makefile was 0600 and after I changed it to 0644 build succeeded without any error. Thank you for pointing it out.
